schema/data/dev/product.sql

```diff
@@ -23,8 +23,8 @@ values
 (22, 'Hot dip galvanizing','tonnes of wire processed','published' ,true, true, false, false),
 (23, 'Hydrogen Peroxide (H2O2)', 'tonnes of pure (100%) Hydrogen Peroxide produced','published' ,true, true, false, false),
 (24, 'Lead-Zinc Smelting','tonnes of lead and zinc','published',true, true, false, false),
-(25, 'Sweet Gas Plants','e3m3OE','published', true, true, false, false),
-(26, 'Sour Gas Plants','e3m3OE','published', true, true, false, false),
+(25, 'Processing sour gas - oil equivalent','cubic metres oil-equivalent','published', true, true, false, false),
```

Comment:
```suggestion
(25, 'Processing sour gas : Oil equivalent','cubic metres oil-equivalent','published', true, true, false, false),
```
...just kidding :laughing: